model/Core/Classes/CreationInfo.md

```diff
@@ -10,6 +10,8 @@ Provides information about the creation of the Element.
 
 The CreationInfo provides information about who created the Element, and when and how it was created. 
 
+The dateTime created is often the date of last change (e.g., a git commit date), not the date when the SPDX data was created, as doing so supports reproducible builds.
+
```

Comment:
CreationInformation is used in a number of use cases, some of which are applicable to reproducable builds, some of which are not.

I would suggest changing the wording to something like:

`If you are producing an SPDX document intended for use in a reproducible build, the dateTime created should be the time of the last change to the element (e.g., a git commit date), otherwise the date when the SPDX data was created should be used.